core/services/ocr2/plugins/ccip/internal/ccipdb/price_service.go

```diff
@@ -46,19 +46,14 @@ const (
 	// 10 min provides sufficient buffer for the Commit plugin to withstand transient price update outages, while
 	// surfacing price update outages quickly enough.
 	priceExpireSec = 600
```

Comment:
@matYang I think this `priceExpireSec` worked as a guard and wiped out everything from the database that was older than 10 minutes. Deleting is no longer the case, so we can enforce that on a query level `where created_at > now() - priceExpireSec`. It will prevent stale prices from being retrieved from the database. 
On the other hand, do we need that check? Is it better to return stale prices or empty dataset?